src/utils/alignment-D3-support.js

```diff
@@ -1,9 +1,21 @@
-// D3 Alignment test supporting functions. Anything that is chart specific
-// should go here. For example, the getTickValueMonth function is only used for
-// testing the Heatmap D3 Chart, so it belongs here.
-
```

Comment:
I thought these deleted comments would be useful to help other devs understand what belongs here. Is there a reason to delete them?

Reply:
Changed the wording in latest push.